tests/tests_release.yml

```
@@ -131,6 +131,7 @@
               insecure: "{{ lsr_rhc_test_data.candlepin_insecure }}"
             rhc_baseurl: "{{ lsr_rhc_test_data.baseurl | d(omit) }}"
             rhc_release: {"state":"absent"}
+            rhc_state: "reconnect"
```

```suggestion
            rhc_state: reconnect
```
https://github.com/redhat-cop/automation-good-practices/tree/main/coding_style#yaml-and-jinja2-syntax
"
Do not use quotes unless you have to, especially for short module-keyword-like strings like present, absent, etc.
"